test/utils.js

```diff
@@ -259,3 +274,42 @@ function makeSettlement (type, balance, overwrites = {}) {
     hash: uuidV4()
   }, overwrites)
 }
+
+async function insertReferralInfos (client) {
+  const ratesPaths = [{
+    path: filePath('0010_geo_referral', 'groups.csv'),
```

Comment:
You can also keep the seed data in an insert query so you don't have to keep a csv file, etc if you prefer